### PR TITLE
Use runUnsynchronized for android_views test

### PR DESCRIPTION
**Pull request by GaryQian**
Synchronized driver wait may be the culprit behind flakes. Try unsynchronized to see if the flakiness improved.

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -2272,7 +2272,6 @@ targets:
     scheduler: luci
 
   - name: Linux android views
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/99001
```
Comment (CaseyHillers) [inline .ci.yaml:2275]: Is this blocking anyone? The flaky test policy is to have the bot remove this label

Comment (GaryQian) [reply .ci.yaml:2275]: This is temporary to make sure the test runs on CI to verify this change works as intended. I'll remove this soon.

Comment (GaryQian) [reply .ci.yaml:2275]: Ok, removed.

**Comment by GaryQian** [comment]
https://ci.chromium.org/p/flutter/builders/try/Linux%20android%20views Runs 6216-6237 are all passing reruns of this PR.

**Review by blasten — APPROVED** [review]

**Pull request merged**